liquibase-core/src/main/java/liquibase/change/core/AddLookupTableChange.java

```diff
@@ -95,8 +97,7 @@ public void setExistingColumnName(String existingColumnName) {
         this.existingColumnName = existingColumnName;
     }
 
-
-    @DatabaseChangeProperty(since = "3.0")
+    @DatabaseChangeProperty(since = "3.0", description = "Name of the database catalog of the table to create")
```

Comment:
Suggest that we include the word "lookup" in this description and others like it, i.e., "Name of the database catalog to create the lookup table."